win32ss/reactx/ntddraw/d3d.c

```diff
@@ -354,15 +355,16 @@ DWORD
 APIENTRY
 NtGdiD3dValidateTextureStageState(LPD3DNTHAL_VALIDATETEXTURESTAGESTATEDATA pData)
 {
-    PGD_D3DVALIDATETEXTURESTAGESTATE pfnD3dValidateTextureStageState = (PGD_D3DVALIDATETEXTURESTAGESTATE)gpDxFuncs[DXG_INDEX_DxD3dValidateTextureStageState].pfn;  
-   
+    PGD_D3DVALIDATETEXTURESTAGESTATE pfnD3dValidateTextureStageState =
+      (PGD_D3DVALIDATETEXTURESTAGESTATE)gpDxFuncs[DXG_INDEX_DxD3dValidateTextureStageState].pfn;
```

Comment:
same here, either do not touch it, or use 2x4, not 2x3
```suggestion
        (PGD_D3DVALIDATETEXTURESTAGESTATE)gpDxFuncs[DXG_INDEX_DxD3dValidateTextureStageState].pfn;
```